Rakefile

```diff
@@ -10,3 +10,20 @@ namespace :gem do
     system 'gem build trollolo.gemspec'
   end
 end
+
+# rake task for rspec and rubocop
+# rake -T (for rake tasks)
+require 'rubocop/rake_task'
+require 'rspec/core/rake_task'
+
+task default: %i[rubocop spec]
+
+desc 'Run tests'
+task :spec do
+  RSpec::Core::RakeTask.new
+end
+
+desc 'Run rubocop'
+task :rubocop do
+  RuboCop::RakeTask.new
```

Comment:
this is not including the same options documented in [the contributing guide](https://github.com/openSUSE/trollolo/blob/master/CONTRIBUTING.md) and used in Travis. It should be the same.